clients/python/README.md

```diff
@@ -72,7 +100,9 @@ model = registry.register_model(
 ### Importing from Hugging Face Hub
 
 To import models from Hugging Face Hub, start by installing the `huggingface-hub` package, either directly or as an
-extra (available as `model-registry[hf]`).
+extra (available as `model-registry[hf]`). Reference section "[installing extras](#installing-extras)" above for
+more information.
```

Comment:
addresses the comment from https://github.com/kubeflow/model-registry/issues/370